netlogo-gui/src/test/ide/JumpToDeclarationTests.scala

```diff
@@ -0,0 +1,97 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.ide
+
+import org.nlogo.core.{Token, TokenType}
+import org.scalatest.FunSuite
+
+class JumpToDeclarationTests extends FunSuite {
+
+  val source1 =
+    """
+    |me-own[
+    |  variable
+    |]
+    |
+    |you-own [
+    |  variable
+    |]
+    |
+    |globals [scared]
+    |to foo
+    |  scared 1
+    |  let scared
+    |  set scared 0
+    |end
+    |
+    |to new [scared]
+    |  set variable 7
+    |  set scared 5
+    |end
+    """.stripMargin
+
+  val source2 =
+    """
+      |to foo
+      |[
+      |  let scared
+      |  set scared 0
+      |]
```

Comment:
Brackets immediately after a procedure name indicate arguments. This may give you the result you want in your tests, but it isn't representative code. `ask [ let scared ... ]` is correct code and would test the appropriate scoping rules.